src/commands/cmd_explain.c

```diff
@@ -59,11 +64,15 @@ void Graph_Explain(void *args) {
 	ExecutionPlan_PreparePlan(plan);
 	ExecutionPlan_Init(plan);       // Initialize the plan's ops.
 
-	if(ErrorCtx_EncounteredError()) goto cleanup;
+	if (ErrorCtx_EncounteredError()) {
+		query_ctx->status = QueryExecutionStatus_FAILURE;
+		goto cleanup;
+	}
 
 	ExecutionPlan_Print(plan, ctx); // Print the execution plan.
 
 cleanup:
+	Info_IncrementNumberOfQueries(&gc->info, query_ctx->flags, query_ctx->status);
```

Comment:
The definition is "Query counts since server start".
I don't really care if GRAPH.EXPLAIN is counted; the effect would be zero or minimal during normal production usage.